packages/lib-classifier/src/components/Classifier/components/withKeyZoom/README.md

```diff
@@ -0,0 +1,33 @@
+# withKeyZoom
+
+A decorator that connects a component to the pan and zoom actions in the `SubjectViewer` store, which handles the image toolbar functions. The wrapped component is given an `onKeyDown` handler but focus management is the responsibility of the component author, not this <abbr title="Higher-Order Component">HOC</abbr>.
```

Comment:
A non-blocking comment: I'm thinking about renaming it `SubjectToolbar` since now it's wired up with the scatter plot, so once that happens, this should reference the `subject toolbar` rather than `image toolbar`.

Reply:
Maybe something about how there's a filter on allowed html tags?

Reply:
Yes, I believe that's why.

Reply:
>I'm thinking about renaming it SubjectToolbar since now it's wired up with the scatter plot, so once that happens, this should reference the subject toolbar rather than image toolbar.

Good point but I don't know if we're going to use this same toolbar for video and audio. The zoom and pan functions seem like they are specific to images and charts.

Reply:
True, but it's not always going to be only pan/zoom controls here. The subject color invert button is going to be added to it otherwise I'd suggest `PanZoomToolbar`. See #1716 

Audio and video may indeed have their own controls that that take the same position in the layout. This is a future discussion though.